.travis.yml

```diff
@@ -0,0 +1,29 @@
+language: cpp
+sudo: false
+
+env:
+  global:
+    - NODE_VERSION="6.1.0"
+
+os:
+  - linux
+  - osx
+
+before_install:
+  - rm -rf ~/.nvm
+  - git clone https://github.com/creationix/nvm.git ~/.nvm
+  - source ~/.nvm/nvm.sh
+  - nvm --version
+  - nvm install $NODE_VERSION
+  - node --version
+  - npm --version
+  - pip install --user cpplint
+
+install:
+  - npm install --build-from-source
```

Comment:
We might want to use the `node_js` base with addons, instead of rolling it all manually, i.e:

```yml
sudo: false
language: node_js
node_js:
  - 6
before_install:
  - pip install --user cpplint
script:
  - npm test
compiler: clang-3.6
env:
  - CCACHE_TEMPDIR=/tmp/.ccache-temp
  - CCACHE_COMPRESS=1
  - CC="clang"
  - CXX="clang++"
addons:
  apt:
    sources:
      - ubuntu-toolchain-r-test
    packages:
      - libstdc++-5-dev
```